src/vl-alert.js

```diff
@@ -106,7 +106,11 @@ export class VlAlert extends vlElement(HTMLElement) {
   };
 
   _titleChangedCallback(oldValue, newValue) {
-    this._titleElement.textContent = newValue;
+    const currentSlot = this.querySelector('[slot="title"]');
+    if (currentSlot) {
+      currentSlot.remove();
+    }
+    this.append(this._template(`<span slot='title'>${newValue}</span>`));
   };
```

Comment:
Deze logica heb ik gereverted. Gedrag is nu dat een slot voorrang krijgt op een attribuut, wat me prima lijkt.